openedx/openedx-platform
# fix: rewrite calcualtor tips to reflect the supported features

## Description and Related information

  It replicates this PR: openedx/frontend-app-learning/pull/825 which
  rewrite calcualtor tips since some tips are no longer relevant,
  hence: #17368

## Testing: 

To be to test this change, here are the steps: 
- Make sure there is a course with `Show Calcutrator` to `true`, follow this [doc link for more details](https://edx.readthedocs.io/projects/open-edx-building-and-running-a-course/en/latest/exercises_tools/calculator.html)  
-  Go to the learning experience as shown (from the above link as well)** 
-  When clicking on `i` button the calculator instructions tips, should reflect the change of this PR. e.g., there should be no example of using T as constant...etc.

** You would need to make sure that you are interacting with the course through the old course experience, **not using the learning MFE** hence, it's already resolved for the learning MFE openedx/frontend-app-learning/pull/825. 
Relative information about how enable use old courseware experience: 
> The courseware.use_legacy_frontend and course_home.course_home_use_legacy_frontend Waffle flags can be toggled on (either globally or per-course-run) in order to revert to the Legacy (LMS Django-rendered) courseware experience.
[ref](https://openedx.atlassian.net/wiki/spaces/COMM/pages/2603816700/Maple)
